doc/presentations/2019-11-12/main.tex

```diff
@@ -0,0 +1,50 @@
+\documentclass{beamer}
+\usepackage[utf8x]{inputenc}
+\usepackage{graphicx}
+\usepackage{tikz}
+\graphicspath{{./../../img/}}
+
+\usepackage[outputdir=latex.out]{minted}
+\usepackage[T1]{fontenc}
+
+\usetheme{TUM}
+
+\title[Research Graph]{Research Graph}
+
+\author[Guerin, Schönball, Silva, Zimmerer]{Fiona Guerin, Raphael Schönball, Alvaro Silva, Andreas Zimmerer}
+\institute{Technical University of Munich}
+
+\date{12 November 2019}
+
+% Customize alert
+\setbeamercolor{alerted text}{fg=TUMZusatzfarbeBlau3}
+\setbeamerfont{alerted text}{series=\bfseries}
+
+\begin{document}
+\maketitle
+
+\begin{frame}{Vision}
+\only<1>{\includegraphics{img_02.png}}
+\end{frame}
+
+\begin{frame}{Mockup}
+	Link: https://marvelapp.com/d3c33g7
+\end{frame}
+
+\begin{frame}{Architecture}
+    \only<1>{\includegraphics{img_03.png}}
+\end{frame}
+
+\begin{frame}{Technology Stack}
+    \only<1>{\includegraphics{img_04.png}}
+\end{frame}
+
+\begin{frame}{Github}
+    https://github.com/Jibbow/research-paper-graph
```

Comment:
btw: you can wrap URL into `\url{...}`.
 1. this influences line breaks in a nice way
 2. I think this makes them clickable(?)
 3. I think this also provides some smart escaping of special characters or so :thinking: